core.spec.md

```diff
@@ -431,9 +424,10 @@ Determine the name of the core specification within the document.
 
 It is possible to [rename the core feature](#sec-Renaming-core-itself) within a document. This process determines the actual name for the core feature if one is present.
 
-**Fails** the *Has Schema* validation if there are no SchemaDefinitions in the document
-**Warns** *Extra Schema* for extra SchemaDefinitions in the document after the first
-**Fails** the *Has Core Feature* validation if the core feature is not referenced with a {@core} directive within the document.
+- **Fails** the *Has Schema* validation if there are no SchemaDefinitions in the document
+- **Warns** *Extra Schema* for extra SchemaDefinitions in the document after the first
```

Comment:
I'm just doing reformatting here. Earlier in the process I asked about error handling and Ashi added this fail/warn concept.

Actually, this one is invalid GraphQL (spec says "When using the type system definition language, a document must include at most one schema definition.") so it seems like this should be a strong failure. That leaves only one warning, which is for invalid feature URL. Shall we just make that a failure too and remove the concept of warnings? Certainly simpler.

Reply:
Yeah. Hm, I could've sworn I found the same thing you just did last time I looked - but didn't find it when I looked today. In that case I'd definitely opt for all fails and removing this validation since it's extraneous. This has some cascading changes to go along with it (the algorithm below is the only thing that comes right to mind).